ansible_builder/steps.py

```diff
@@ -1,6 +1,26 @@
 import os
 
 from . import constants
+from .colors import MessageColors
+
+
+class AdditionalBuildSteps:
+    def __init__(self, additional_steps):
+        """Allows for additional prepended / appended build steps to be
+        in the Containerfile or Dockerfile.
+        """
+        self.steps = []
+        if isinstance(additional_steps, str):
+            lines = additional_steps.strip().splitlines()
+        elif isinstance(additional_steps, list):
+            lines = additional_steps
+        else:
+            lines = []
+            print(MessageColors.WARNING + "Unknown type found for additional_build_steps.  Ignoring..." + MessageColors.ENDC)
```

Comment:
my own preference would be failure.

As the volume of output grows, it becomes less and less likely that the user will see this warning. The only way to make sure they see it is to refuse to produce the output.

Reply:
Changed!